swiftide/src/indexing/node.rs

```diff
@@ -57,9 +60,16 @@ impl Debug for Node {
             .field("chunk", &self.chunk)
             .field("metadata", &self.metadata)
             .field(
-                "vector",
-                &self.vector.as_ref().map(|v| format!("[{}]", v.len())),
+                "vectors",
+                &self
+                    .vectors
+                    .iter()
+                    .map(HashMap::iter)
+                    .flatten()
+                    .map(|(embed_type, vec)| format!("'{embed_type}': {}", vec.len()))
```

Comment:
Logs will be a bit more verbose - instead of vector length it will be _metadata1: len1, metadata2: len2_

Reply:
A disclaimer comment.
Resolving it to calm down PR merging rules.